### ISSUE TITLE
Student attendance interface a little confusing

### ISSUE THREAD

**Issue by symbyte** [colegarien/SDDTermProject#80]
Looking at the new graph, I am just confused about what the # column represents. Is it the number of absenses, the number of attendances or the number of the class session?

 In the next sprint if you could clarify the way the table is labeled to make it obvious what data the table is displaying that would be great. Please let me know if you have any questions by leaving a message here!

**Comment by telecom2009** [comment]
This is number of class session

**Comment by ralhamami** [comment]
Yeah I did have a similar issue. I've started testing updates in both 18 and Marshmallow now. 

The # column was just stating what class period it was in the semester. So if it was the third class that semester, it would say 3. Hope that helps. :+1:

**Comment by symbyte** [comment]
I wonder if the class session number even needs to be there since there is also a date, its basically giving the same information in two ways, so its sort of redundant. I dont mind the info staying there, just change the label on the table so it would be more clear what that column means to an end-user

**Comment by ralhamami** [comment]
Yeah I can definitely understand where you're coming from there, even though this isn't my module. I will probably refrain from implementing a similar column in my version of it.
